src/style.css

```diff
@@ -1 +1,25 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins. It's better to add margin to only one side (top or bottom) to avoid potential margin collapse issues.